src/Dfe.ContentSupport.Web/Controllers/HomeController.cs

```diff
@@ -5,14 +5,13 @@
 
 namespace Dfe.ContentSupport.Web.Controllers;
 
-public class HomeController(IContentfulService contentfulService)
-    : Controller
+public class HomeController(IContentfulService contentfulService) : Controller
 {
     public async Task<IActionResult> Index(string slug)
     {
         if (string.IsNullOrEmpty(slug)) return RedirectToAction("error");
+        var resp = await contentfulService.GetContent(slug) as ContentSupportPage;

```

Comment:
You might want to stay away from unnecessary casting here - have the method signature return the object type you want unless it's intended to be generic (in say, some kind of overload)

Reply:
@jack-coggin